app/views/exchanges/employer_applications/_index.html.erb

```diff
@@ -22,7 +22,7 @@
         </tr>
       </thead>
       <tbody>
-        <% benefit_applications.order('start_on DESC').each do |application| %>
+        <% benefit_applications&.order('start_on DESC').each do |application| %>
```

Comment:
Leaving off the safe operator causes an issue if there is only one benefit application